src/pfe/portal/modules/metricsService/index.js

```diff
@@ -25,12 +25,16 @@ const deepClone = (obj) => JSON.parse(JSON.stringify(obj));
 const metricsCollectorInjectionFunctions = {
   nodejs: nodeMetricsService.injectMetricsCollectorIntoNodeProject,
   liberty: injectMetricsCollectorIntoLibertyProject,
+  // Open Liberty currently identifies as docker
+  docker: injectMetricsCollectorIntoLibertyProject,
   spring: injectMetricsCollectorIntoSpringProject,
 }
 
```

Comment:
can we add to this that its java and docker that openliberty is rather than just docker?

Reply:
Fixed by https://github.com/eclipse/codewind/pull/1962/commits/317c67ef694f76706aad6a27d9275a573ad9e931